File: vapor-app/src/main/java/com/tevinjeffrey/vapor/okcloudapp/ProgressListener.java
package com.tevinjeffrey.vapor.okcloudapp;

/**
 * Listener for progress
 */
public interface ProgressListener {

    /**
     * When progress is reported
     * @param current The current progress
     * @param max The max progress amount
     */
    void onProgress(final long current, final long max);
}

File: vapor-app/src/main/java/com/tevinjeffrey/vapor/okcloudapp/ProgressListener.kt
package com.tevinjeffrey.vapor.okcloudapp

/**
 * Listener for progress
 */
interface ProgressListener {

    /**
     * When progress is reported
     * @param current The current progress
     * *
     * @param max The max progress amount
     */
    fun onProgress(current: Long, max: Long)
}